examples/democoin/mock/validator.go

```diff
@@ -44,7 +44,7 @@ func (v Validator) GetDelegatorShares() sdk.Dec {
 }
 
 // Implements sdk.Validator
-func (v Validator) GetRevoked() bool {
+func (v Validator) GetJailed() bool {
```

Comment:
I'd change it to `IsJailed()`

Reply:
the pattern is `GetXYZ()` with all the other functions

Reply:
I don't disagree with this particular one but it's probably better not to be inconsistent with the rest